_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -14,6 +15,14 @@
 
 
 // ----------------------------
+// #PATHS
+// ----------------------------
+$static-path: '..' !default;
+$font-path: '#{$static-path}/fonts' !default;
+$image-path: '#{$static-path}/images' !default;
+
```

Comment:
Are you making a separate variable for `$static-path` in case it changes? I feel like, at least like this, the "dot dot" as a variable is redundant and might be better included in the `$font-path` and `$image-path` variables.